packages/suite/src/components/suite/modals/ReduxModal/DeviceContextModal/PassphraseModal.tsx

```diff
@@ -128,14 +127,15 @@ export const PassphraseModal = ({ device }: PassphraseModalProps) => {
                                 <Translation
                                     id="TR_PASSPHRASE_DESCRIPTION_ITEM1"
                                     values={{
-                                        a: chunks => (
```

Comment:
Changes in this file are not needed. Just change text to `"It's important to first <a>learn how a passphrase works</a>."`